crates/re_space_view_tensor/src/visualizer_system.rs

```diff
@@ -20,25 +18,11 @@ impl IdentifiedViewSystem for TensorSystem {
     }
 }
 
-struct TensorVisualizerEntityFilter;
-
-impl VisualizerAdditionalApplicabilityFilter for TensorVisualizerEntityFilter {
-    fn update_applicability(&mut self, event: &re_data_store::StoreEvent) -> bool {
-        diff_component_filter(event, |tensor: &re_types::components::TensorData| {
-            !tensor.is_vector()
-        })
-    }
-}
-
 impl VisualizerSystem for TensorSystem {
     fn visualizer_query_info(&self) -> VisualizerQueryInfo {
         VisualizerQueryInfo::from_archetype::<Tensor>()
     }
 
-    fn applicability_filter(&self) -> Option<Box<dyn VisualizerAdditionalApplicabilityFilter>> {
-        Some(Box::new(TensorVisualizerEntityFilter))
-    }
```

Comment:
Heuristics changes this close to release 😬  Let's test this very carefully

Reply:
not a heuristic change but a applicability/visualizability change. As pointed out by a a user this one has a "clever workaround" by first logging a 2d tensor and then making it 1D. We once identified it then as applicable tensor and then

Reply:
ah nice that's now also described in #5839